kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -70,8 +69,195 @@
     </KGrid>
 
     <hr class="bottom-border">
+    <div v-if="isQuestionAvailable">
+      <KGrid>
+        <KGridItem
+          :layout12="{ span: 6 }"
+        >
+          <div class="left-column-alignment-style">
+            <div class="align-kcheckbox-style">
+              <p>
+                <KCheckbox />
+              </p>
+            </div>
+
+            <div>
+              <p>{{ $tr('selectAllLabel') }}</p>
+            </div>
+          </div>
+        </KGridItem>
+
+        <KGridItem
+          :layout12="{ span: 6 }"
+        >
+          <div class="right-alignment-style">
+            <KGrid>
+              <KGridItem :layout12="{ span: 4 }">
+                <div class="icon-container">
+                  <KIcon class="reduce-chervon-spacing" icon="chevronDown" />
+                  <KIcon class="reduce-chervon-spacing" icon="chevronUp" />
+                </div>
+              </KGridItem>
+
+              <KGridItem :layout12="{ span: 4 }">
+
+                <KIcon
+                  class="icon-size"
+                  icon="refresh"
+                />
+              </KGridItem>
+
+              <KGridItem :layout12="{ span: 4 }">
+                <KIcon
+                  class="icon-size"
+                  icon="trash"
+                />
+              </KGridItem>
+            </KGrid>
+          </div>
+        </KGridItem>
+
+      </KGrid>
+
+      <AccordionContainer
+        :items="placeholderList"
+      >
+        <template
+          #default="{ isItemExpanded, toggleItemState }"
+        >
+          <div
+            v-for="item in placeholderList"
+            :key="item.id"
+          >
+            <AccordionItem
```

Comment:
In `AccordionItem` the whole thing is wrapped in a `Draggable` component and the focus is going there but I'm not really sure it should.

When focusing the "Select all" checkbox it goes:

- Draggable (w/ no core outline)
- Drag handle
- KCheckbox
- The `<a>` that toggles opening/closing the section
-- Then on to the next row...

I think that this can be fixed by adding `tabindex="-1"` to the `AccordionItem` here so that it can be focused, but is bypassed during keyboard navigation.

Reply:
Ah I had a thought occur about this and the dragging setup as it is altogether. Will follow-up on Slack with more details